Dockerfile

```diff
@@ -13,15 +11,16 @@ ENV HOME /home/lounge
 ENV LOUNGE_HOME ${HOME}/data
 ENV LOUNGE_SRC ${HOME}/src
 
-RUN mkdir -p ${LOUNGE_SRC}
-WORKDIR ${LOUNGE_SRC}
-
-RUN curl -L https://github.com/thelounge/lounge/archive/v${LOUNGE_VERSION}.tar.gz | tar --strip-components 1 -xzf -
-
-RUN chown -R lounge:lounge ${LOUNGE_SRC}
+# Create a non-root user for lounge to run in.
+RUN useradd --create-home --uid 1000 lounge
```

Comment:
Is it recommended to specify the user ID?! I understand the need to create a non-root user, but I feel like we should leave the details up to the OS, no?